client/jetpack-cloud/sections/partner-portal/company-details-form/style.scss

```diff
@@ -42,3 +42,11 @@
 		}
 	}
 }
+
+.company-details-form__tos p {
+	margin: 45px 0 24px;
+	font-weight: 400;
+	font-size: 1rem;
+	line-height: 24px;
+	color: #000000;
```

Comment:
We could replace `#000#` with `var( --studio-black );` but I dno how strict we are with it 😅